client/src/pages/LoginPage.jsx

```diff
@@ -0,0 +1,108 @@
+import { useState, useEffect } from "react";
+import loginModel from "/src/assets/loginmodel.png";
+
+const LoginPage = () => {
+  const [width, setWidth] = useState(window.innerWidth);
+  useEffect(() => {
+    const handleResize = () => {
+      setWidth(window.innerWidth);
+    };
+    window.addEventListener('resize', handleResize);
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+  return (
+    <div className="h-screen w-screen bg-gradient-to-b from-color4 to-gray-400 flex justify-center items-center">
+      <div className="h-2/3 w-5/6 flex justify-center items-center">
+        <div className="p-8 rounded">
+          <h1 className="text-white font-poppins text-6xl font-bold mb-2 pb-8">
+            Welcome Back!
+          </h1>
+          <p className="text-white font-poppins text-xl mb-6">
+            Please enter your login information
+          </p>
+
+          <form>
```

Comment:
Add a `LoginForm` component. As there will be a good amount of code for that, It will be easier for future changes. For now just add the whole form tag

This will address: #12